filetests/isa/x86/extractlane-run.clif

```diff
@@ -0,0 +1,31 @@
+test run
+set enable_simd
+
+function %test_extractlane_b8() -> b8 {
+ebb0:
+    v0 = bconst.b8 true
+    v1 = splat.b8x16 v0
+    v2 = extractlane v1, 10
+    return v2
+}
+; run
+
+function %test_extractlane_i16() -> b1 {
+ebb0:
+    v0 = vconst.i16x8 0x00080007000600050004000300020001
+    v1 = extractlane v0, 1
+    v2 = icmp_imm eq v1, 2
+    return v2
+}
+; run
+
+function %test_extractlane_f32() -> b1 {
+ebb0:
+    v0 = f32const 0x42.42
+    v1 = splat.f32x4 v0
```

Comment:
Can we use a vconst.f32x4 instead and only put the float value in the one lane that is read later? (otherwise even if it read the wrong lane, the test would be correct.)